src/View.js

```diff
@@ -0,0 +1,257 @@
+
+            return (
+              <Paneset id="eresources-paneset">
+                {filterPaneIsVisible &&
+                  <Pane
+                    defaultWidth="20%"
+                    id="pane-eresources-search"
+                    paneTitle={<FormattedMessage id="stripes-smart-components.searchAndFilter" />}
+                  >
+                    <form onSubmit={onSubmitSearch}>
+                      <div className={css.searchGroupWrap}>
+                        <FormattedMessage id="ui-plugin-find-eresource.searchInputLabel">
+                          {ariaLabel => (
+                            <SearchField
+                              aria-label={ariaLabel}
+                              autoFocus
+                              className={css.searchField}
+                              data-test-eresource-search-input
+                              id="input-eresource-search"
+                              inputRef={searchField}
+                              marginBottom0
+                              name="query"
+                              onChange={getSearchHandlers().query}
+                              onClear={getSearchHandlers().reset}
+                              value={searchValue.query}
+                            />
+                          )}
+                        </FormattedMessage>
+                        <Button
+                          buttonStyle="primary"
+                          disabled={!searchValue.query || searchValue.query === ''}
+                          fullWidth
+                          id="clickable-search-eresources"
+                          marginBottom0
+                          type="submit"
+                        >
+                          <FormattedMessage id="stripes-smart-components.search" />
+                        </Button>
+                      </div>
+                      <div className={css.resetButtonWrap}>
+                        <Button
+                          buttonStyle="none"
+                          disabled={disableReset()}
+                          id="clickable-reset-all"
+                          onClick={resetAll}
+                        >
+                          <Icon icon="times-circle-solid">
+                            <FormattedMessage id="stripes-smart-components.resetAll" />
+                          </Icon>
+                        </Button>
+                      </div>
+                      <Filters
+                        activeFilters={activeFilters.state}
+                        data={data}
+                        filterHandlers={getFilterHandlers()}
+                        showPackages={showPackages}
+                      />
+                    </form>
+                  </Pane>
+                }
+                <Pane
+                  appIcon={<AppIcon app="agreements" />}
+                  defaultWidth="fill"
+                  firstMenu={(filters) => {
```

Comment:
I believe this is out of sync with current UX guidelines. We got behind with how the show/hide filters buttons work and I had to update it in Agreements recently, so use that to inform you here.

(May as well do it in the other plugins as well while you're at it.)